### DISCUSSION TITLE
Window pivot?

### DISCUSSION THREAD

**Discussion by mystal** [emilk/egui#134] [Q&A] [answered]
Hi! I am porting my game from using `imgui-rs` to `egui` and was wondering how I could achieve the following in `egui`:

```rust
Window::new(im_str!("Debug Stats"))
    .position([0.0, screen_height], Condition::Always)
    .position_pivot([0.0, 1.0])
```

Which positions the bottom left corner of the `imgui-rs` window on the bottom left corner of my game's screen. It looks like this in-game:
![image](https://user-images.githubusercontent.com/74258/105643914-962aac00-5e47-11eb-8cc8-2b854302baad.png)

Does this feature currently exist in `egui`? If so, how can I achieve it?

Thanks!

**Comment by emilk** [comment, marked as answer]
No, this does not currently exists. It would be a very useful thing to have though, so please create a feature-request issue for it!

**Reply by mystal** [reply]
Thanks for the reply, @emilk!

I created an issue: https://github.com/emilk/egui/issues/138
